src/modules/populateTasks.js

```diff
@@ -1,48 +1,54 @@
-import createHTMLelement from './createHTMLelement.js';
-
-let tasks = [
-  {
-    description: 'Sneak peak: To Do list',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Pro-tips about your IDE',
-    completed: false,
-    index: 2,
-  },
-
-  {
-    description: 'Morning session: Peer-to-peer code review',
-    completed: true,
-    index: 3,
-  },
-
-  {
-    description: 'Daily standup',
-    completed: true,
-    index: 4,
-  },
-
-  {
-    description: 'Pro-tips about your IDE',
-    completed: false,
-    index: 5,
-  },
-
-  {
-    description: 'Get better at seeking help',
-    completed: true,
-    index: 6,
-  },
-];
-
-const tasksContainer = document.querySelector('.todolist-placeholder');
-
-tasks = tasks.sort((a, b) => (b.index - a.index));
-
-export default () => {
-  tasks.forEach((task) => {
-    createHTMLelement('li', 'task-item', 'task-item', `${task.description}`, tasksContainer);
+import myTaskList from './TaskList.js';
+import { createHTMLElement, createInputElement } from './createHTMLelement.js';
+
+const populateTasks = () => {
+  const tasksContainer = document.querySelector('.todolist-placeholder');
+  const sortedTasks = myTaskList.tasks.sort((a, b) => (b.index - a.index));
+  tasksContainer.innerHTML = '';
+  const taskItem = createHTMLElement('li', 'task-item', 'task-item', '', tasksContainer);
+  const taskInput = createInputElement('input', 'new-task text-inp', 'new-task', 'text', 'new-task', '', taskItem);
+  taskInput.placeholder = 'Add to your list';
+  const addTaskBtn = createHTMLElement('button', 'add-task', 'add-task', '<i class="fa fa-arrow-circle-left" aria-hidden="true"></i>', taskItem);
+
+  addTaskBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    const newTask = document.querySelector('.new-task');
+    myTaskList.addTask(newTask.value, false, myTaskList.tasks.length + 1);
+    populateTasks();
   });
-};
\ No newline at end of file
+
+  sortedTasks.forEach((task) => {
+    const taskElement = createHTMLElement('li', 'task-item', 'task-item', '', tasksContainer);
+    createInputElement('input', 'task-complete', 'task-complete', 'checkbox', 'task-complete', '', taskElement);
+    createInputElement('input', 'task-edit text-inp', 'task-edit', 'text', 'task-edit', `${task.description}`, taskElement);
+    const taskEdit = createHTMLElement('button', 'task-edit', `${task.index}`, '<i class="fa fa-arrows" aria-hidden="true"></i>', taskElement);
+    const taskDrag = createHTMLElement('button', 'task-drag', 'task-drag', '<i class="fa fa-ellipsis-v" aria-hidden="true"></i>', taskElement);
+    const taskDelete = createHTMLElement('button', 'task-delete', `${task.index}`, '<i class="fa fa-trash-o" aria-hidden="true"></i>', taskElement);
+
+    taskDelete.addEventListener('click', (e) => {
+      e.preventDefault();
+      myTaskList.removeTask(taskDelete.id);
+      console.log(myTaskList);
```

Comment:
- [ ] kindly make sure you remove the ` console.log();` from your codebase for JavaScript **[best practices.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)** It’s ugly, it kills performance and it can make confidential data visible to anyone using the browser tools to look at your website.

Reply:
Oh, thanks for the suggestion. I will actually remove the file since it was not used